web/Services/LandlordService.cs

```diff
@@ -71,4 +73,16 @@ await using (var transaction = await _dbContext.Database.BeginTransactionAsync(I
 
         return ILandlordService.LandlordRegistrationResult.Success;
     }
+
+    public List<PropertyViewModel> GetListOfProperties(int landlordId)
+    {
+
+        var landlord = _dbContext.Landlords
+            .Include(l => l.Properties)
+            .SingleOrDefault(l => l.Id == landlordId);
+        // Assumes landlord does exist

```

Comment:
Then use `.Single`, not `.SingleOrDefault`. I'd also probably either:

* Change this to query Properties directly
* Or have the method return the landlord (i.e. `GetLandlordWithProperties`)